ercc/Makefile

```diff
@@ -14,6 +14,33 @@ decorator-plugin:
 test:
 	go test -v
 
-clean:
+clean: docker-clean
 	go clean
 
+PEER_NAME?=dev-jdoe
+ERCC_NAME?=ercc
+DOCKER_CONTAINER=$(PEER_NAME)-$(ERCC_NAME)-0
+DOCKER_IMAGE?=$$(docker images | grep -- -$(ERCC_NAME)- | awk '{print $$1;}')
+
+docker:
+	docker build -t $(DOCKER_IMAGE) .
+
+docker-run:
+	docker run \
+                -it \
+                --device /dev/isgx \
```

Comment:
why do we need access to sgx in ercc?

Reply:
mhh ... just wondering why this docker build for ercc at all?

Reply:
was copy/paste, so definitely no need for sgx.  Definitely wanted the docker-clean aspect of it (which i've also added to ecc) but thought for dev mode a docker run could be useful. But thinking of it it actually not really  useful. will remove